### DISCUSSION TITLE
Rescan images periodically

### DISCUSSION THREAD

**Discussion by bgoareguer** [aquasecurity/starboard#416] [Development]
Starboard scans images when containers are created.

If a container runs for a long period of time, it would be nice that Starboard periodically triggers a rescan to ensure that the image is not affected by newly published vulnerabilities.

The rescan interval should be configurable with a default value of 24h for example.

**Comment by danielpacak** [comment]
👋 @bgoareguer Sorry for late reply. We'd like to add such functionality to Starboard. As you pointed out it's important to not only scan new or updated workloads but rescan existing workloads on a regular basis. We have #537 to track such enhancement.
